win32ss/gdi/ntgdi/freetype.c

```diff
@@ -1612,7 +1611,7 @@ FillTMEx(TEXTMETRICW *TM, PFONTGDI FontGDI,
     TM->tmOverhang = 0;
     TM->tmDigitizedAspectX = 96;
     TM->tmDigitizedAspectY = 96;
```

Comment:
Can you refresh my memory: what does this "96" value correspond to? (it is also used later in the code).

Reply:
96 is a typical DPI (dots per inch) resolution value that was widely used in Windows. If DPI is higher, it means that the display pixels are finer. DPI can be changed by user. I'll use a macro named 'DPI'.

Reply:
I replaced "96"'s with "DEFAULT_DPI" macros.